### PR TITLE
[chore-5383]: refactor hidden input and PlainTexts to TS

### PR DESCRIPTION
**Pull request by vdegraaf**
Ticket: [SIG-5383](https://gemeente-amsterdam.atlassian.net/browse/SIG-5383)

### CODE REVIEW

File: src/signals/incident-management/components/HiddenInput/index.tsx [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
+
 import type { FC } from 'react'
 
 interface HiddenInputProps {
   name: string
   value: string
 }
 
+/**
+ * @deprecated
```
Comment (OscarBakker) [inline src/signals/incident-management/components/HiddenInput/index.tsx:17]: why is this added? should it be removed?

File: src/signals/incident/components/form/types/PlainText.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export interface PlainTextMeta {
+  type?: string
+  value: string
+  name: string
+  isVisible: boolean
+  label?: string
+  ignoreVisibility?: boolean
+  // valueAuthenticated is undefined in Wizard step 1. Not sure where/how it is used.
```
Comment (OscarBakker) [inline src/signals/incident/components/form/types/PlainText.ts:8]: good one, don't know either

File: src/signals/incident/components/form/types/PlainText.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export interface PlainTextMeta {
+  type?: string
```
Comment (OscarBakker) [inline src/signals/incident/components/form/types/PlainText.ts:2]: optionals should be on last lines

File: src/signals/incident/components/form/types/FileInput.ts [hunk 4]
```diff
@@ -19,15 +19,21 @@ export interface Meta extends FormMeta {
...
-  meta: {
-    addToSelection?: () => void
-    getClassification?: () => void
-    handleSubmit?: () => void
-    incidentContainer?: () => void
-    removeFromSelection?: () => void
-    updateIncident: (data: any) => void
-    wizard?: WizardSection
+export interface ParentMeta {
+  addToSelection?: () => void
```
Comment (OscarBakker) [inline src/signals/incident/components/form/types/FileInput.ts:23]: optionals last

File: src/signals/incident/components/form/PlainText/PlainText.test.tsx [hunk 5]
```diff
@@ -123,52 +128,52 @@ describe('Form component <PlainText />', () => {
...
         type: 'info',
-      })
+      }
 
-      render(withAppContext(<PlainText {...props} />))
+      render(withAppContext(<PlainText {...defaultProps} meta={customMeta} />))
 
       expect(screen.getByTestId('plain-text')).toBeInTheDocument()
-      expect(screen.getByText(props.meta.value)).toBeInTheDocument()
+      expect(screen.getByText(defaultProps.meta.value)).toBeInTheDocument()
```
Comment (OscarBakker) [inline src/signals/incident/components/form/PlainText/PlainText.test.tsx:139]: yep good distinction to use default and custom props, although props instead of customProps, should be more like the other tests

File: src/signals/incident-management/components/HiddenInput/index.tsx (hunk 1 above)
Comment (vdegraaf) [reply src/signals/incident-management/components/HiddenInput/index.tsx:17]: It's added to prevent someone using it. If someone tries to use it they will get a "this component is deprecated" warning.

**Review by OscarBakker — APPROVED** [review]
